app/v1/controllers/requestParsers/validators/validations/ConsolidatedExpensesValidation.scala

```diff
@@ -39,4 +40,24 @@ object ConsolidatedExpensesValidation {
       }
     }
   }
+
+  def validateCreate(expenditure: ForeignPropertyExpenditure, path: String): List[MtdError] = {
```

Comment:
Since the create and amend models contain mostly the same fields, could both endpoints share models? ie, each endpoint has its own `XXXForeignPropertyRequestBody` model but use the same inner models? It would get rid of a lot of unnecessary duplication.

Reply:
It might be worth refactoring this now it's working so it's neater; code duplication is just asking for tedious maintenance (assuming the models are going to stay mirrored, which is the current assumption we're making)

Reply:
So the packages would look something like this:
<img width="318" alt="Screenshot of new folder structure" src="https://user-images.githubusercontent.com/31282758/86927599-6f1ac000-c12b-11ea-9e4a-69d92b5ffcab.png">
(name them how you want to)

Reply:
Sure sounds good, would it be better to do this in this pr or make a seperate pr for a model refactor?

Reply:
I guess it can be done separately, I'll make a ticket to get that done and merge this in.